tests/test_enterprise_access.yaml

```diff
@@ -23,6 +23,19 @@
     - method: 'POST'
     - expected_status: [400]
 
+- test:
+    - name: 'Assignment configuration list endpoint returns HTTP 200'
+    - url: '/enterprise/v1/assignment-configurations/9101d3de36156cba/admin/assignments/'
+    - method: 'GET'
+    - headers: { 'Authorization': 'aeiou', 'Content-Type': 'application/json' }
+    - expected_status: [200]
+
+- test:
+    - name: 'Assignment configuration list request rejected without authorization'
+    - url: '/enterprise/v1/assignment-configurations/0123456789abcdefg/admin/assignments/'
+    - method: 'GET'
+    - expected_status: [400]
+
```

Comment:
can we add one more test with any of the valid query params and then one with an invalid param?

Reply:
Sure 🫡

Reply:
@sameenfatima78 added one more test please have a look